packages/react-tabs/.storybook/decorators/withThemeProvider.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ThemeProvider } from '@fluentui/react-next';
+import { useCustomizationOptions } from '../knobs/theme';
+
+export const withThemeProvider = storyFn => {
```

Comment:
Hmm we should make these shared too since now they are being duplicated in 2 places. `@uifabric/build` package has a folder called `storybook`, i think that's a good place to put

Reply:
Logged #14144 to move storybook files to `@uifabric/build`.